Library/Homebrew/cmd/update.sh

```diff
@@ -375,6 +375,13 @@ user account:
 EOS
   fi
 
+  # we may want to use Homebrew CA certificates
+  if [[ -n "${HOMEBREW_FORCE_BREWED_CA_CERTIFICATES}" && ! -f "${HOMEBREW_PREFIX}/etc/ca-certificates/cert.pem" ]]
+  then
+    # we cannot install Homebrew CA certificates if homebrew/core is unavailable.
+    ([[ -d "${HOMEBREW_LIBRARY}/Taps/homebrew/homebrew-core" ]] && brew install ca-certificates) || true
```

Comment:
```suggestion
    ([[ -d "${HOMEBREW_LIBRARY}/Taps/homebrew/homebrew-core" || -n "$HOMEBREW_INSTALL_FROM_API" ]] && brew install ca-certificates) || true
```
Maybe?

Reply:
Looks like that makes sense, though the curl and git stuff don't have this yet. cc @Rylan12